.codegen/resolvers.go.tmpl

```diff
@@ -0,0 +1,37 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package resolvers
+
+import (
+	"context"
+    "fmt"
+	"github.com/databricks/cli/bundle"
+)
+
+type ResolverFunc func(ctx context.Context, b *bundle.Bundle, name string) (string, error)
+
+func Resolvers() map[string](ResolverFunc) {
+    resolvers := make(map[string](ResolverFunc), 0)
+    {{range .Services -}}
+    {{- if not .IsAccounts -}}
```

Comment:
Conversely, is there an `IsWorkspace` that we can use?

Reply:
@pietern unfortunately not